project_task_stage_change_restriction/tests/test_stage_change_restriction.py

```
@@ -0,0 +1,301 @@
+            {"name": "Allowed Stage Group (T)"}
+        )
+
+        def _make_user(login, extra_groups):
+            """
+            Create a test user and assign core, extra and internal-user groups.
+
+            :param str login: system login for the new user
+            :param list extra_groups: list of res.groups records to grant
+            :return: newly created res.users record
+            """
+            groups = extra_groups + [cls.grp_internal]
```

Ruff (0.11.9)

Consider `[*extra_groups, cls.grp_internal]` instead of concatenation

Replace with `[*extra_groups, cls.grp_internal]`

(RUF005)